test/options.test.js

```diff
@@ -67,6 +67,33 @@ exports.mappingOverSomeAppliesFunctionToValue = function(test) {
     test.done();
 };
 
+
+exports.flatMapOnNoneToNone = function(test) {
```

Comment:
I wonder if just putting some pseudo-JavaScript would make the test clearer e.g.

```javascript
exports(none.flatMap(func) is none)
```